### ISSUE TITLE
How activate the plugin, looks like not workgin ?

### ISSUE THREAD

**Issue by gamersalpha** [wbernest/mattermost-plugin-rssfeed#15]
System : ubuntu 18.04 lts
mattermost : mattermost-5.12.2-linux-amd64.tar.gz
rssfeed install : i use a git clone from today (07/07/2019 10:30)(i can't found a .gz stable realease by mattermost version ?)

so i can see the plugin in system console : 
![image](https://user-images.githubusercontent.com/32466950/60766173-fa3f2100-a0a5-11e9-8c4c-9318c10d213d.png)

but when i try to lunch the command : 

`/feed help`
nothing happening : 

![image](https://user-images.githubusercontent.com/32466950/60766178-1642c280-a0a6-11e9-85a3-6d670ac0557b.png)

and if i force the message send nothing come : 

![image](https://user-images.githubusercontent.com/32466950/60766182-265aa200-a0a6-11e9-8fb8-cfbf66a5ee8e.png)

what do u think is wrong please

**Comment by wbernest** [comment]
It looks like it is not running.  You can activate the plugin from the plugin management screen.  If it does not continue running after your start it, you will need to check your mattermost logs and see what error is being logged.

**Comment by mthld** [comment]
hi,

I got the same problem, the plugin won't stay activated and keep being disable itself.
<img width="388" alt="Screenshot 2019-08-03 at 14 43 09" src="https://user-images.githubusercontent.com/41020854/62412074-65c0d380-b5fd-11e9-86b1-db28ec41b447.png">

```
2019-08-03T14:42:46.680+0200	error	mlog/log.go:172	An error occurred while trying to execute this command.	{"path": "/api/v4/commands/execute", "request_id": "ty9g69b3ptgdby73b3mhgyzk3o", "ip_addr": "185.94.189.187", "user_id": "rjq8m341y3btie17ju4ha3w3rw", "method": "POST", "err_where": "ExecutePluginCommand", "http_code": 500, "err_details": "err=plugin not found: rssfeed"}
```

<img width="438" alt="Screenshot 2019-08-03 at 14 43 54" src="https://user-images.githubusercontent.com/41020854/62412077-6c4f4b00-b5fd-11e9-9556-d4639b59e211.png">

When I enable it again I get the following error:
```
2019-08-03T14:46:14.446+0200	error	mlog/sugar.go:23	Failed to create RSSFeed Plugin. Attempting to find existing one.	{"plugin_id": "rssfeed", "err": "SqlUserStore.Save: An account with that email already exists., user_id=i5z9j5jmb3dafr6iec7us6kgme, pq: duplicate key value violates unique constraint \"users_email_key\""}
```

It's the same problem already mentioned in #14, and I think #13 is the source of it.

Thanks in advance @wbernest for your help.

**Comment by aberenyi** [comment]
I doesn't work for me either, however the logs are showing a different error after a disable-enable cycle:

```
{"level":"error","ts":1574090826.2624626,"caller":"plugin/client_rpc_generated.go:67","msg":"RPC call OnConfigurationChange to plugin failed.","plugin_id":"rssfeed","error":"connection is shut down"}
```

/cc @wbernest @levb

**Comment by mthld** [comment]
FYI the problem I described in this thread was fixed by #23

**Comment by aberenyi** [comment]
Ignore my [previous](https://github.com/wbernest/mattermost-plugin-rssfeed/issues/15#issuecomment-555066258) comment, there was a version mismatch.
